src/vscommand.ts

```diff
@@ -42,7 +42,10 @@ function displayResult(result: string | undefined): void {
 export async function registerCommands(...modules: string[]): Promise<Disposable[]> {
     // this step is required to get all annotations processed and this is drawback,
     // because it triggers javascript loading when extension is activated
-    await Promise.all(modules.map((module) => import(module)));
+    await Promise.all(modules.map(async (module) => {
+            try {await import(module);} catch { /* Ignore */ }
```

Comment:
This is effectively silencing any errors caused by importing the modules.

When it's bundled, the modules shouldn't need to be imported. Is there a way that we can avoid trying to import the modules when bundled?

I think it's bad that we are silencing all import errors here.

Reply:
If we don't process these errors, the activation failure gets "effectively silenced" and never ends without any valuable reason, so the extension stay in `Activating...` until restarted.

Actually these errors ALWAYS  happen  when running the extension installed from VSIX as it physically does not have any of the specified `modules` present. Imho, these errors are expected in production and it's better to hide these errors rather then have them always to be reported in production

Reply:
I think it would be better to skip trying to import all the modules in production, what do you think about this?

Reply:
Yes, but do we have any indication of either we run in production or not?

Reply:
I don't know, do you know of a way?

Reply:
When I started, I was expecting that we don't have to use `import (module)` at all before registering the commands, because I explicitly call `moduleXXX.getInstance()` before the commands registration. But this didn't work for some reason.
Now, as everything looks like working now, we probably can just remove that list of paths of the modules to be imported - they all already referenced ( === 'inported').

Reply:
And yes. This actually works.